service/util/meta.json

```diff
@@ -1,4 +1,5 @@
-[{
+{
```

Comment:
Is meta.json the same as metaAndSchema.json?
Does metaAndSchema.json only used for test purpose? And you probably need to remove "metadata" and "schema" keys from meta.json, otherwise it will failed to load meta from server.go